zulint/custom_rules.py

```diff
@@ -25,12 +25,10 @@ def __init__(
         self,
         langs: Sequence[str],
         rules: Sequence[Rule],
-        shebang_rules: Sequence[Rule] = [],
```

Comment:
Do you think it's worth adding a shebang line check to `example_lint` or the documentation? I feel like `\A` constructions being what they want may not be fast for others to figure out :).

Reply:
Meh—it’s basically fine if users end up using `^`. But if you feel like adding more documentation, go for it.